<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>

<head>
  <title>SwissPedHealth</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
  <!--[if lte IE 8]><script src="/assets/js/ie/html5shiv.js"></script><![endif]-->
  <link rel="stylesheet" href="/assets/css/main.css" />
</head>


<body>

    <!-- Wrapper -->
    <div id="wrapper">

        <!-- Main -->
        <div id="main">
            <div class="inner">

                <!-- Header -->
  <header id="header">
    <a href="/" class="logo"><strong>SwissPedHealth</strong> by SPHN & PHRT
	</a>


    <ul class="icons"> 
      <li><a href="" class="icon"><span class="label">Home</span></a></li>
	  <li><a href="about.html" class="icon"><span class="label">About</span></a></li>
	  <li><a href="/#news" class="icon"><span class="label">News</span></a></li>
	  <li><a href="/study_overview" class="icon"><span class="label">Projects</span></a></li>
	  <li><a href="/#contact" class="icon"><span class="label">Contact</span></a></li>

	  <!-- To remove label text for incons, twiter,github,etc: see `.icon > .label` in assets/css/main.css --> 
      
      
      
      
      
      
      
      
      
      
    </ul>
  </header>


	  <div class="logos"style="margin-top:10px; margin-bottom:-30px">
		    <!-- <a href=""> -->
			    <img src="assets/images/logos/PHRT_Logo_transparent_50px.png" alt="Logo image" width="150" height="38.5" style="text-decoration: none; margin-bottom:0px"/>
			  <!-- </a> -->
		    <!-- <a href=""> -->
			    <img src="assets/images/logos/SPHN-Logo-SIB-Colors-155x111.png" alt="Logo image" width="93" height="66" style="text-decoration: none; margin-bottom:-10px" />
		  <!-- </a> -->
	  </div>


                <!-- Content -->
                <section>
                	<header class="main">
                		<h1>Lighthouse project</h1>
                	</header>

                  

                <h2 id="development-and-validation-of-a-multi-omics-workflow-to-diagnose-rare-disease">Development and validation of a multi-omics workflow to diagnose rare disease</h2>
<!-- ## Background -->

<div class="row">
	<div class="6u 12u$(small)">
		<h3>Life-threatening Phenotypes</h3>
Rare diseases are a group of over 7,000 disorders that have a strong impact on health and have major social and economic consequences.
They affect 400 million people worldwide, including about 500,000 in Switzerland.
Rare diseases often affect children, causing premature death or chronic disability.
They are found at increased rates in critically ill neonates or children admitted to ICUs.
Rare diseases can cause life-threatening conditions such as metabolic dysfunction, encephalopathy, severe infection, or cardiac failure.
Extreme phenotypes (severe forms of rare diseases) are interesting targets for finding rare, causal genetic and biochemical variants because patients are more likely to carry harmful low-frequency or de novo alleles.
The identification of causal genetic and biochemical variants in rare, severe diseases can be extremely valuable for patients, particularly children. 
By gaining a better understanding of these diseases, healthcare professionals may be able to develop more effective treatments and provide better care for affected individuals.
</div>

<div class="6u 12u$(small)">
<h3> Promise of Multi-omics for Personalized Medicine</h3>
In the past decade, the discovery rate of the causes of rare diseases has been increasing due to the ability to generate and analyze genomic data at reasonable cost.
A pioneering project in Australia used rapid DNA sequencing (WES) and resulted in a genetic diagnosis in 51% of children, leading to a change in management in 76% of patients.
Adding transcriptomic data (RNA-seq) increases the diagnostic yield of WES or WGS.
Multi-omics technologies simultaneously assess the genome, transcriptome, proteome, and metabolome, which provides a better understanding of genotype-phenotype relationships.
Integration of multi-omics data has the potential to identify rare genomic variants and their functional validation, leading to a higher diagnostic yield compared to WGS alone.
Multi-omics has the potential to be a breakthrough diagnostic approach in children with extreme phenotypes by enabling the prompt initiation of lifesaving treatments and/or a change in management.
</div>
</div>

<hr />

<h2 id="gaps-in-current-practice">Gaps in current practice</h2>

<p>WES and WGS are used in diagnostics, but patient selection is biased and there are still a high proportion of patients with undiagnosed diseases. 
The identification of large numbers of Variants of Unknown Significance (VUS) in DNA sequencing datasets makes the picture more complicated.
There is a need for a multi-omics workflow to provide better diagnostic yield compared to WGS.</p>

<p>Current gaps in knowledge and practice include:</p>

<ol>
  <li>Delayed diagnosis of rare diseases.
There is often a delay in diagnosing rare diseases and inconsistencies in investigating and treating them.</li>
  <li>Lack of novel approaches to rare disease diagnosis.
Conventional methodologies (WES and biochemical testing) are used, but they can be time consuming and have low diagnostic yield.</li>
  <li>Unavailability of clinical multi-omics.
Multi-omics technologies are currently limited to research settings.</li>
  <li>Clinical diagnostic workflows not adapted for multi-omics.
Clinical diagnostic workflows are not designed for discovering novel diseases.</li>
  <li>Lack of information from end-users on multi-omics.
Information on how providers and families respond to multi-omics results is currently unknown.</li>
</ol>

<hr />

<h2 id="project-phases">Project phases</h2>

<p><span style="color: green; font-size: 150%;">☑</span>
<strong>Phase 1 Complete:</strong>
Multi-omics approach was used to diagnose and characterize 210 individuals affected with the rare inborn error of metabolism methylmalonic aciduria.
The combination of genomic, transcriptomic, proteotyping, and metabolomic information with phenotypic dataset led to the identification of a molecular diagnosis for 84% of the patients.
The multi-omics technology was found to be powerful for diagnosis and discovery of pathomechanistic factors in rare diseases.
A workflow for data acquisition, transfer, storage, and analysis within the PHRT/SPHN infrastructure was established.
Another genomic and transcriptomic workflow was established to identify 2 novel diseases in 89 patients with extreme immunological phenotypes.
This resulted in identifying rare diseases in children with life-threatening extreme phenotypes through combining clinical evaluation and multi-omics exploration.</p>

<p><span style="color: green; font-size: 150%;">☐</span>
<strong>Phase 2 Ongoing:</strong>
The integrated approach of WGS, RNA-seq, proteotyping, and metabolomics will be applied to existing cohorts of extreme phenotypes to identify rare deleterious genetic variants.
Novel computational methods will be developed to integrate multi-omics data into a diagnostic workflow.</p>

<p><span style="color: green; font-size: 150%;">☐</span>
<strong>Phase 3 Ongoing:</strong>
The workflow developed in Phase 2 will be tested in a prospective cohort of 140 children with extreme phenotypes.
Provider and family uptake of multi-omics information will be examined.
The aim is to identify rare diseases causing extreme phenotypes even in single patients.</p>

<hr />

<h2 id="research-design">Research Design</h2>

<h3 id="patient-recruitment-and-phenotype-selection-responsible-group-1">Patient Recruitment and Phenotype Selection (Responsible: Group 1)</h3>

<p><strong>Phase 2:</strong>
Access existing cohorts of 250 children with extreme phenotypes where consent to use stored biospecimens is available.
Children with severe metabolic dysfunction (n=150):
(i) Biochemically diagnosed patients without molecular data
(ii) Patients with VUS (Variants of Uncertain Significance)
(iii) Undiagnosed patients.
Children suspected of primary immunodeficiency (n=100):
(i) Severe infection in the absence of clinical risk factors
(ii) Severe immune dysregulation (Pediatric Inflammatory Multisystem Syndrome associated with SARS-CoV-2).</p>

<p><strong>Phase 3:</strong>
Multi-site study in a prospective cohort of 140 patients admitted to or referred to intensive care.
Blood samples (PAXgene for RNA extraction, EDTA for DNA extraction, leukocytes/serum for proteome/metabolome assessment), skin biopsies, and mucosal swabs from family members will be obtained where feasible.
A panel of consortium members will review and prioritize cases.
Inclusion criteria:
(i) Acute encephalopathy (ruled out trauma, infection, intoxication, and asphyxia).
(ii) Life-threatening metabolic derangement.
(iii) Unusually severe infection.
(iv) Primary cardiac failure.
(v) Out-of-hospital sudden infant death or near-fatal event.
(vi) Unusually severe inflammation syndromes (cytokine storm syndrome, hemophagocytic lymphohistiocytosis, COVID-19-associated Pediatric Inflammatory Multisystem Syndrome).
(vii) Undiagnosed diseases (if not already included in Phase 2).</p>

<h3 id="clinical-data-processing-responsible-group-2">Clinical Data Processing (Responsible: Group 2)</h3>
<p>Implement core SwissPedData set to capture clinical features and outcomes.
Test SwissPedData with EHR data, SRDR and MDSi to characterize phenomes.
Embed the lighthouse project within SwissPedHealth to:
(i) Gather essential clinical data of the included patients.
(ii) Enrich SwissPedData with MDSi and SRDS.
(iii) Demonstrate how NDS (National Data System) can be enhanced with high resolution omics data.
(iv) Establish a whole-of-life data stream to capture future outcomes.
(v) Ascertain the incidence of identified rare diseases.</p>

<h3 id="multi-omics-data-generation--responsible-group-3">Multi-omics Data Generation  (Responsible: Group 3)</h3>

<ul>
  <li>
    <p>Phase 2 and Phase 3: Patient-derived Biospecimens:
Patient-derived biospecimens from Phase 2 and Phase 3 will be sent to the Swiss Multi-Omics Center (SMOC).</p>
  </li>
  <li>
    <p>Nucleic Acid-Based Analyses:
(i) WGS and RNA-seq data will be generated and analyzed by the Health 2030 Genome Center (GC).
(ii) The unaligned DNA and RNA sequence data is clinical grade and can be used for clinical decision making.</p>
  </li>
  <li>
    <p>Proteomic Profiling:
(i) Performed by the Clinical Proteotype Analysis Center (CPAC).
(ii) Uses next-generation proteomics technology (DIA-MS).
(iii) Converts a clinical specimen into a digital proteotype map.</p>
  </li>
  <li>
    <p>High-Throughput Targeted/Untargeted Metabolomics and Lipidomics Analyses:
(i) Performed by the Clinical Metabolomics Analysis Center (CMAC).
(ii) Analysis of primary metabolites and xenobiotics is done by high-throughput flow injection, time-of-flight mass spectrometry.</p>
  </li>
  <li>
    <p>Data Processing and Sharing:
(i) All centers are part of the Swiss Multi-Omics Center (SMOC).
(ii) SMOC is integrated into the SPHN/BioMedIT secure computational infrastructure.
(iii) Provides secure data processing and sharing.</p>
  </li>
</ul>

<h3 id="primary-data-analyses-responsible-group-4">Primary Data Analyses (Responsible: Group 4)</h3>

<p>Automated DNA variant calling and annotation to search for novel disease-causing variants.
Prioritization of rare variants based on:
(i) Potentially damaging effect.
(ii) Known function of the gene in the clinical presentation.
(iii) Degree of purifying selection.
RNA-seq data analysis for quantitative expression profiling and alternative splicing analysis.
Coordinated integration and visualization of DNA variants, RNA transcripts, proteoforms, and metabolites.
Mapping onto biological networks and pathways using tools developed by SMOC and the Fellay group at EPFL.
Data storage and analysis performed within SPHN BioMedIT infrastructure to ensure privacy, security, and adherence to FAIR principles.
Downstream analyses using multimodal machine learning to integrate multi-omics and clinical data.</p>

<h3 id="multi-omics-results-using-multimodal-machine-learning-analysis-responsible-group-5">Multi-omics results using multimodal machine learning analysis (Responsible: Group 5)</h3>

<p><strong>Phase 2:</strong>
A machine learning approach will be used to analyze multi-omics data and clinical phenotype data.
A cohort of 250 children with extreme phenotypes and fully annotated multi-omics data will be used to train a multimodal model.
Multi-view clustering will be used to identify features that are most important for the prediction.
New methods suited for rare disease diagnostics will be developed for data with a low sample size but high dimensionality.
A novel probabilistic approach to multimodal constrained clustering will be derived to guide the algorithm towards a desirable configuration.
The validated machine learning workflow will be used to identify relevant features within a complex multi-omics dataset.</p>

<p><strong>Phase 3:</strong>
The validated machine learning workflow will be utilized in a prospective cohort.
The workflow will be optimized for singleton diagnosis using simultaneously obtained whole blood specimens.
The model output will provide a probability scale of multi-omics data explaining the observed extreme phenotype, aligned with ACMG recommendations.
Enrichment with trio or kindred WGS data, or cell-derived functional datasets will be possible.</p>

<h3 id="implementation-of-findings-responsible-group-6">Implementation of findings (Responsible: Group 6)</h3>

<p>Study information will be fed back from the research setting into clinics.
Diagnostic yield will be defined as the primary outcome.
The study results will be reviewed by an expert panel.
Information on potentially relevant genetic findings will not be released if not related to a rare disease causing the extreme phenotype.
Collaboration with the undiagnosed disease network and care4rare will occur where appropriate.</p>

<h3 id="bioethical-acceptability-and-implementation-from-end-users-responsible-group-7">Bioethical acceptability and implementation from end-users (Responsible: Group 7)</h3>

<p><strong>Phase 1:</strong> Literature Review and Clinical Experiences.
Review of the literature on attitudes towards the offer and return of results from WES or WGS in the rare disease setting in countries outside of Switzerland.
Clinical experiences of associated investigators Ormond and Rauch (medical geneticist University of Zurich) who have been providing predictive and diagnostic genetic testing results for decades.
PPI-focused discussions with patients and family members who have undergone previous clinical omics testing.</p>

<p><strong>Phase 2:</strong> Exploratory Interviews.
Up to 25 medical providers and 25 parents will be interviewed in English, German or French to determine their attitudes towards a hypothetical offer and return of results.
The interview guide will be developed based on the findings from phase 1.
Interviews will be transcribed, coded, and analyzed.</p>

<p><strong>Phase 3:</strong> Identification of Information Needs and Best Practices.
Identify the information families need to consider in order to make informed decisions about the returned results.
Determine how best to facilitate discussions with families and medical providers.
Identify a process to return results that maximizes descence and autonomy and minimizes risks of potential harm.
Nest this work with the PPI work done as part of SwissPedHealth WP 4102.</p>

<hr />

<h2 id="scientific-output-and-expected-contributions">Scientific Output and Expected Contributions</h2>

<div class="row">
<div class="6u 12u$(small)">
<h3>Development of a Novel Multi-Omics Workflow </h3>
<p>
The lighthouse project will deliver a multi-omics workflow optimized for rare disease diagnosis.
This workflow goes beyond the current state-of-the-art of WES/WGS and will deliver genetic, molecular, and functional information.
</p>
</div>

<div class="6u 12u$(small)">
<h3>
Identification of Novel Diagnoses and Pathomechanisms 
</h3>
<p>
Multi-omics analysis of existing and prospective cohorts will identify novel diagnoses and pathomechanisms for rare diseases.
Machine learning will be implemented to integrate the data and demonstrate its usefulness as a diagnostic tool.
</p>
</div>
</div>

<div class="row">
<div class="6u 12u$(small)">
<h3>
Standardized Framework for Rare Disease Diagnostics 
</h3>
<p>
The provider and patient perspective will be included to provide a standardized framework for all pediatric Swiss research hubs.
The prospective cohort will focus on extreme phenotypes predominantly recruited in intensive care settings, but the approach is scalable to other conditions and settings.
The multi-omics workflow will be combined with the SwissPedHealth dataset to obtain information on incidence, clinical criteria, and outcomes.
The SwissPedHealth data stream will be used to test data-driven strategies to pre-select patients most likely to benefit from multi-omics.
The lighthouse project will demonstrate the capacity of the SwissPedHealth data stream to integrate research projects with high density omics data.
</p>
</div>

<div class="6u 12u$(small)">
<h3>
Accelerate Discovery in Rare Disease Diagnostics and Therapeutics 
</h3>
<p>The goal of the lighthouse project is to set a sustainable framework for translational studies on children with the aim to accelerate discovery in rare disease diagnostics and therapeutics.</p>
</div>
</div>

<!-- <h2 id="content">Summary content</h2> -->
<!-- <p>Praesent ac adipiscing ullamcorper semper ut amet ac risus. Lorem sapien ut odio odio nunc. Ac adipiscing nibh porttitor erat risus justo adipiscing adipiscing amet placerat accumsan. Vis. Faucibus odio magna tempus adipiscing a non. In mi primis arcu ut non accumsan vivamus ac blandit adipiscing adipiscing arcu metus praesent turpis eu ac lacinia nunc ac commodo gravida adipiscing eget accumsan ac nunc adipiscing adipiscing lorem ipsum dolor sit amet nullam veroeros adipiscing.</p> -->

<!-- <h2 id="content">Main content</h2> -->
<!-- <p>Praesent ac adipiscing ullamcorper semper ut amet ac risus. Lorem sapien ut odio odio nunc. Ac adipiscing nibh porttitor erat risus justo adipiscing adipiscing amet placerat accumsan. Vis. Faucibus odio magna tempus adipiscing a non. In mi primis arcu ut non accumsan vivamus ac blandit adipiscing adipiscing arcu metus praesent turpis eu ac lacinia nunc ac commodo gravida adipiscing eget accumsan ac nunc adipiscing adipiscing lorem ipsum dolor sit amet nullam veroeros adipiscing.</p> -->

<!-- <div class="row"> -->
<!-- 	<div class="6u 12u$(small)"> -->
<!-- 		<h3>Shorter content</h3> -->
<!-- 		<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat commodo eu sed ante lacinia. Sapien a lorem in integer ornare praesent commodo adipiscing arcu in massa commodo lorem accumsan at odio massa ac ac. Semper adipiscing varius montes viverra nibh in adipiscing blandit tempus accumsan.</p> -->
<!-- 	</div> -->
<!-- 	<div class="6u$ 12u$(small)"> -->
<!-- 		<h3>Shorter content</h3> -->
<!-- 		<p>In arcu accumsan arcu adipiscing accumsan orci ac. Felis id enim aliquet. Accumsan ac integer lobortis commodo ornare aliquet accumsan erat tempus amet porttitor. Ante commodo blandit adipiscing integer semper orci eget. Faucibus commodo adipiscing mi eu nullam accumsan morbi arcu ornare odio mi adipiscing nascetur lacus ac interdum morbi accumsan vis mi accumsan.</p> -->
<!-- 	</div> -->
<!-- 	<div class="4u 12u$(medium)"> -->
<!-- 		<h3>Shorter content</h3> -->
<!-- 		<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat eu sed ante lacinia sapien lorem accumsan varius montes viverra nibh in adipiscing blandit.</p> -->
<!-- 	</div> -->
<!-- 	<div class="4u 12u$(medium)"> -->
<!-- 		<h3>Shorter content</h3> -->
<!-- 		<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat eu sed ante lacinia sapien lorem accumsan varius montes viverra nibh in adipiscing blandit.</p> -->
<!-- 	</div> -->
<!-- 	<div class="4u$ 12u$(medium)"> -->
<!-- 		<h3>Shorter content</h3> -->
<!-- 		<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat eu sed ante lacinia sapien lorem accumsan varius montes viverra nibh in adipiscing blandit.</p> -->
<!-- 	</div> -->
<!-- </div> -->



              </div>
          </div>

          <!-- Sidebar -->
  <div id="sidebar">
    <div class="inner">

      <!-- <!-1- Search -1-> -->
      <!--   <section id="search" class="alt"> -->
      <!--     <form method="post" action="#"> -->
      <!--       <input type="text" name="query" id="query" placeholder="Search" /> -->
      <!--     </form> -->
      <!--   </section> -->

      <!-- Menu -->
        <nav id="menu">
          <header class="major">
            <h2>Menu</h2>
          </header>
          <ul>
            <li><a href="/">Homepage</a></li>
            <li><a href="about.html">About</a></li>
            <!-- <li><a href="patient_checklist.html">Patient checklist</a></li> -->
            <li>
              <span class="opener">Projects</span>
              <ul>
                <li><a href="lighthouse_project.html">Lighthouse</a></li>
                <li><a href="nested_project_1.html">Nested project 1</a></li>
                <li><a href="nested_project_2.html">Nested project 2</a></li>
                <li><a href="nested_project_3.html">Nested project 3</a></li>
				<li><a href="nested_project_4.html">Nested project 4</a></li>
                <li><a href="study_overview.html">Study overview</a></li>
              </ul>
            </li>
            <li><a href="/">Data privacy</a></li>
            <li><a href="/">Clinic information</a></li>
            <li>
              <span class="opener">Research methods</span>
              <ul>
                <li><a href="research_methods_statistical_analysis.html">Statistical analysis</a></li>
                <li><a href="research_methods_multi_omics.html">Multi-omics</a></li>
                <li><a href="/">Infection analysis</a></li>
                <li><a href="/">Cancer research</a></li>
                <li><a href="/">Anthropometric analysis</a></li>
                <li><a href="/">Electronic health record</a></li>
                <li><a href="/">Clinical data warehouse</a></li>
              </ul>
            </li>
            <li><a href="our_team.html">Our team</a></li>
            <li><a href="/">Funding</a></li>
            <li><a href="/#news">In the news</a></li>
          </ul>
        </nav>

      <!-- Section -->
        <section>
          <header class="major">
            <h2>Highlights for patients</h2>
          </header>
          <div class="mini-posts">
            <article>
              <a href="#" class="image"><img src="assets/images/national-cancer-institute-W6yy0wYV-hk-unsplash.jpg" alt="" /></a>
              <p>Genomic analysis allows our scientists to identify health concerns and discover new treatments.</p>
            </article>
            <article>
              <a href="#" class="image"><img src="assets/images/charlesdeluvio-pjAH2Ax4uWk-unsplash.jpg" alt="" /></a>
              <p>Machine learning is used to detect causes of disease more rapidly than traditional approaches and supplements the routine basic care which is provided in clinic.</p>
            </article>
            <article>
              <a href="#" class="image"><img src="assets/images/andriyko-podilnyk-a-4U35u31IQ-unsplash.jpg" alt="" /></a>
              <p>Pediatric health promotes care during development, to provide a safe and secure environment, and have opportunities for learning starting from birth.</p>
            </article>
          </div>
          <ul class="actions">
            <li><a href="#" class="button">More</a></li>
          </ul>
        </section>

      <!-- Section -->
        <section>
          <header class="major">
            <h2>Get in touch</h2>
          </header>
		  <p>For clinic appointments please consult your physician. For research inquiries please use the contact information provided. For patient involvement please contact xyz@email.</p>
          <ul class="contact">
            <li class="fa-envelope-o"><a href="#">swisspedhelth@proton.me</a></li>
            <li class="fa-phone">+41 21 693 18 74</li>
            <li class="fa-home">Haus der Akademien Laupenstrasse 7<br />
            Bern,  3001<br />
            Switzerland</li>
          </ul>
        </section>

      <!-- Footer -->
        <footer id="footer">
          <p class="copyright">&copy; SwissPedHealth. All rights reserved. Images: <a href="https://unsplash.com/license">license</a>. Design: <a href="https://html5up.net">HTML5 UP</a> Production: <a href="https://lawlessgenomics.com">Dylan Lawless</a>.</p>
        </footer>

    </div>
  </div>


      </div>

      <!-- Scripts -->
  <script src="assets/js/jquery.min.js"></script>
  <script src="assets/js/skel.min.js"></script>
  <script src="assets/js/util.js"></script>
  <!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
  <script src="assets/js/main.js"></script>


</body>

</html>
